gateway/slack.go

```diff
@@ -119,6 +123,31 @@ func NewSlackClient() *SlackClient {
 	}
 }
 
+func (sc *SlackClient) getConversations(gcp *slack.GetConversationsParameters) (channels []slack.Channel, cursor string, err error) {
+	var rateLimitErr *slack.RateLimitedError
+	var ok bool
+
+	for retries := 0; retries < getConvoRetries; retries++ {
```

Comment:
am I reading this wrong? it looks like it would run through this for loop 3 times if all the calls to `sc.client.GetConversations` succeed and `err == nil`

Reply:
Oops I'm dumb